src/app-profile.ts

```diff
@@ -29,7 +29,7 @@ export interface AppProfileOptions {
    * value is required when creating the app profile and optional when setting
    * the metadata.
    */
-  routing?: 'any' | Cluster;
+  routing?: 'any' | Cluster | Set<Cluster>;
```

Comment:
a thought - it might be nice to provide an additional method which takes in a set of strings - we don't need the cluster object anywhere, so this could be easier on the user rather than having to construct a Cluster object.